science/netcdf/Portfile

```diff
@@ -7,9 +7,7 @@ PortGroup                   cmake 1.0
 PortGroup                   muniversal 1.0
 
 
-github.setup                Unidata netcdf-c 4.6.1 v
-revision                    1
-epoch                       3
```

Comment:
The `epoch` line cannot be removed. It can only increase. Otherwise no one will see when the port get updated.